db-mojo-scorer/README.md

```diff
@@ -0,0 +1,258 @@
+# Database Scorer
+Some customers need to select rows from a Database, score and write back the predictions.Using DAI and then this program enables them to quickly build and use models.
+
+Supports following databases
+1. Postgres DB
+2. Microsoft SQL Server 12
+3. TerraData
+4. RedShift
+5. Oracle
+6. DB2
+7. Azure Data Warehouse
+8. Snowflake
+9. ElasticSearch
+ 
+## Building
```

Comment:
Remove the Build steps as we are not sharing the source publicly, customers get the the jar.